woof-code/rootfs-skeleton/usr/sbin/sfs_load

```diff
@@ -49,7 +49,13 @@
 
 . /etc/rc.d/PUPSTATE
 
-if [ -L /bin -o -L /sbin -o -L /lib -o -L /lib64 -o -L /lib32 -o -L /libx32 ]; then
+sym_list=""
+for fld2 in bin sbin lib lib64 lib32 libx32
```

Comment:
Is it enough? /usr/lib64 can be a symlink, too.

Reply:
Again - another different update - also /usr/lib ??

Reply:
I'll quote you in #3320:

> /lib64 and /usr/lib64 are symlinks

If you're trying to fix the problem for /lib64, why not solve it for /usr/lib64 as well?